tests/ci/docker_images/linux-arm/ubuntu-20.04_accp_base/Dockerfile

```diff
@@ -16,7 +16,6 @@ RUN apt-get install -y build-essential
 RUN apt-get install -y cmake
 RUN apt-get install -y lcov
 RUN apt-get install -y wget
-RUN apt-get install -y clang-format
```

Comment:
?

Reply:
The version in Ubuntu 20 breaks our CI. clang-format is checked in the AL2 CI.